zerver/models.py

```diff
@@ -301,6 +301,10 @@ class Realm(models.Model):
     upload_quota_gb = models.IntegerField(null=True)  # type: Optional[int]
 
     VIDEO_CHAT_PROVIDERS = {
+        'disabled': {
+            'name': u"Disabled",
```

Comment:
Should the label be "disabled", or "None"?  The latter might be clearer given the heading of "Video chat provider"?